### DISCUSSION TITLE
Introducing FluentMigrator in an existing production database

### DISCUSSION THREAD

**Discussion by robertmircea** [fluentmigrator/fluentmigrator#1473] [Q&A] [answered]
What is the recommended way of introducing fluentMigrator for a several years old with lots of data database? 

I know how to script the existing database for the initial migration, but for production I would like just to consider applied the initial migration (not doing any changes to production). I was thinking about support in FluentMigrator for creating only the versioninfo table and baselining the existing database to an existing version. After the baseline version, the newer migrations I would expect to be applied as usual.

Does this such a feature exist in FluentMigrator?

**Comment by Ralf1108** [comment, marked as answer]
Steps:
1. Create empty migration with version number
2. Run empty migration on production system -> VersionInfo table will be created with one entry
3. Now change migration code from step 1 to include your baseline script for your database
This won't do any harm to production database because the database is already up to date!
4. Add additional migrations as you want
